lms/djangoapps/courseware/module_render.py

```diff
@@ -713,7 +713,6 @@ def rebind_noauth_module_to_user(module, real_user):
         wrappers=block_wrappers,
         get_real_user=user_by_anonymous_id,
         services={
-            'i18n': ModuleI18nService(),
             'fs': FSService(),
             'field-data': field_data,
             'user': DjangoXBlockUserService(user, user_is_staff=user_is_staff),
```

Comment:
I think we're going to have to ask @cpennington to provide some more clarity here -- I unfortunately don't have all the context around this requirement, either.